.travis.yml

```
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6" # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+script:
```

Please bring the style checking section the same way we do for other projects
https://github.com/wavefrontHQ/wavefront-sdk-python/blob/master/.travis.yml
https://github.com/wavefrontHQ/wavefront-pyformance/blob/master/.travis.yml
```suggestion
install:
  - pip install flake8 flake8-colors flake8-import-order
  - pip install pep8-naming
  - pip install pydocstyle
  - pip install pylint
before_script:
  - python -m flake8
  - python -m pylint wavefront_sdk
  - python -m pydocstyle
script:
```